<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QlineEdit{
	font-size: 10 px;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>140</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>марка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>модель</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>коробка передач</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальная скорость</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>разгон от 0 -100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пробег</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ОбЪём топливного бака</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>380</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Длинна</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ширина</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Высота</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дорожный просввет</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>количество мест</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Вид</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>90</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Год выпуска</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Участие в  ДТП</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Состояние</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена машины на  момент покупки</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_17">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Citroen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Volkswagen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Opel</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>120</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Ручная</string>
   </property>
   <item>
    <property name="text">
     <string>Ручная</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Автомат</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>270</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Да</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Нет</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>220</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>430</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>320</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string>идеальное</string>
   </property>
   <item>
    <property name="text">
     <string>идеальное</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>нормальное</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ужасное</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
